src/utils/charts/chartFormatting.js

```diff
@@ -1,23 +1,22 @@
 import _ from 'lodash';
 
-/** Helper functions to format vlaues
+/** Helper functions to format vlaues for charts
```

Comment:
"to format vlaues" => "to format *values*"

Reply:
Done! Better?

Reply:
:+1: